src/scss/theme.scss

```diff
@@ -1,4 +1,186 @@
-@import './sphinx-bootstrap';
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400|Lato:400');
+
+html {
+  font-size: 15px;
+}
+
+body {
+  background-color: white;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  line-height: 1.65;
+  color: #333;
+  padding-top: 80px;
+}
+
+p {
+  margin-bottom: 1.15rem;
+
+  // &.rubric {
+  //   border-bottom: 1px solid rgb(201, 201, 201);
```

Comment:
This is no longer needed? 
You can check it at https://144-130237506-gh.circle-artifacts.com/0/html/demo/generated/pandas.DataFrame.drop.html#pandas.DataFrame.drop, where indeed there is still a line under "Examples". But it is strange that this still seems to come from sphinx-bootstrap.css, which should be removed now ..

Reply:
@hoetmaaiers so there still is pandas_sphinx_theme/static/sphinx-bootstrap.css_t that should be removed I think? (since you moved that content into /src/, otherwise it duplicates the css) 
That's also the reason that I still see a correct rendering of the "Examples" title (see my comment above). So the "rubric" css above probably needs to be uncommented.

Reply:
Well spot, this is fixed. 

I was also thinking of using a welback clean plugin. This removes the generated output in `/static`. This makes it more clear to the committing person that `yarn build:production` should be ran or all js & css output is deleted in the commit.